app/components/primer/alpha/live_region_announcement.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= render Primer::BaseComponent.new(**@system_arguments) do %>
+  <template shadowrootmode="open">
+    <style>
+      :host {
+        clip-path: inset(50%);
+        height: 1px;
+        overflow: hidden;
+        position: absolute;
+        white-space: nowrap;
+        width: 1px;
+      }
+    </style>
+    <div id="polite" aria-live="polite" aria-atomic="true"></div>
+    <div id="assertive" aria-live="assertive" aria-atomic="true"></div>
```

Comment:
I noticed the `announce` utility depends on these `id="polite"` and `id="assertive"` being set as it calls `document.querySelector`. If there are multiple `<live-region>` element on a page (e.g. because there is a modal), this results in duplicate IDs. Is this problematic? @joshblack 🤔

Reply:
Hey @khiga8! Are these for the `id="polite"` and `id="assertive"` values specifically? If so, these are in the Shadow DOM and won't conflict with other live regions on the page as a result.